content/developer/accessibility/index.md

```diff
@@ -0,0 +1,254 @@
+
+#### Button or link?
+
+If clicking on this element does something on the page, it's a button. If it goes somewhere, it's a link. 
+
+
+#### Lang
+
+Besides using `lang="en"` on your `<html/>` tag, you should also use it on links to other languages, or any other foreign language elements. Example, if you've got a multilingual site: add it to the links that switch languages.
+
+### Error messages
+
+- `aria-errormessage` contains ID of element with Error message
+- If the message is added after the user has clicked submit or on blur, use `aria-live="assertive"`
+
+Don't overuse `aria-live="assertive"` as it will stop reading anything being read and start reading the new message immediately, but in the case of an important error, it's appropriate.
+
+### Hiding things from the screen reader user
+
+If any information is really irrelevant to a non sighted user, such as an icon that is next to a label that clearly defines that section, you may want to hide it. You can do so by using `aira-hidden="true"`.
+
+### Hiding things from a sighted user
+
+If you want to add additional details to a screen reader user and it's not possible to do so with `aria-label` or `aria-labelledby`, there's a hack that's become a convention that you can use. The traditional name for this class is `sr-only` (screen reader only). 
+
+Here's the class:
+
+```css
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0,0,0,0);
+  border: 0;
+}
+```
+
+Modern screen readers are good at emulating what CSS does. In other words, if you `display: none;` something, it will also be hidden to the screen reader. The above CSS make the element invisible, but it does so by cropping it in a 1px x 1px container. This is a hack, but it's become a convention. Maybe one day we'll have a proper CSS property for this behavior, but we currently don't.
+
+### Focus states
+
+All elements that can have focus need to have a visual indication they are. Default browser stayles already have this, but it's pretty frequent for designers to ask the outline be removed. If you are to remove the default style, just keep in mind that you will need to add it to **EVERY** focussable element. That's a lot more work than keeping the default.
+
+If you put focus on an element it needs to come into view. That means if you've got a carousel and the user tabs away from the visible slides, the slides need to come over. Alternatively, you could set a `tabindex="-1"` on non visible slides to avoid this problem.
+
+### Good Mobile Menu (aka. Hamburger Menu)
+
+- Use a real button
+- For "drawer" style (a.k.a. expand and collapse) sections, use a button!
+- Use `aria-expanded`
+- Use `aria-haspopup`
+- You can tab into it after clicking the menu button
+- Close menu on "blur" and on "esc"
+
+### Responsive design
+
+Responsive design is now a requirement for accessibility. 
+
```

Comment:
It might be a good idea to cover mobile click targets size directive also.

Something like: "Apple’s iPhone Human Interface Guidelines recommends a minimum target size of 44 pixels wide 44 pixels tall. Microsoft’s Windows Phone UI Design and Interaction Guide suggests a touch target size of 34px with a minimum touch target size of 26px. Nokia’s developer guidelines suggest that the target size should be no smaller than 1cm x 1cm square or 28 x 28 pixels."

Reference: https://www.smashingmagazine.com/2012/02/finger-friendly-design-ideal-mobile-touchscreen-target-sizes/